config/metadata/asset_resource.yaml

```diff
@@ -0,0 +1,357 @@
+    index_keys:
+      - "program_title_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  episode_title:
+    type: string
+    multiple: true
+    index_keys:
+      - "episode_title_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  segment_title:
+    type: string
+    multiple: true
+    index_keys:
+      - "segment_title_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  raw_footage_title:
+    type: string
+    multiple: true
+    index_keys:
+      - "raw_footage_title_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  promo_title:
+    type: string
+    multiple: true
+    index_keys:
+      - "promo_title_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  clip_title:
+    type: string
+    multiple: true
+    index_keys:
+      - "clip_title_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  program_description:
+    type: string
+    multiple: true
+    index_keys:
+      - "program_description_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  episode_description:
+    type: string
+    multiple: true
+    index_keys:
+      - "episode_description_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  segment_description:
+    type: string
+    multiple: true
+    index_keys:
+      - "segment_description_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  raw_footage_description:
+    type: string
+    multiple: true
+    index_keys:
+      - "raw_footage_description_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  promo_description:
+    type: string
+    multiple: true
+    index_keys:
+      - "promo_description_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  clip_description:
+    type: string
+    multiple: true
+    index_keys:
+      - "clip_description_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  producing_organization:
+    type: string
+    multiple: true
+    index_keys:
+      - "producing_organization_sim"
+      - "producing_organization_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  admin_data_gid:
+    type: string
+    multiple: false
+    form:
+      required: false
+      primary: false
+      multiple: false
+  series_title:
+    type: string
+    multiple: true
+    index_keys:
+      - "series_title_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  series_description:
+    type: string
+    multiple: true
+    index_keys:
+      - "series_description_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  intended_children_count:
+    type: string
```

Comment:
This needs to include index_keys

Reply:
additional indexing will be handled in another issue